app/controllers/reactions_controller.rb

```diff
@@ -62,6 +63,7 @@ def create
         category: category,
       )
       @result = "create"
+      Moderator::SinkArticles.call(@reaction.user_id) if vomit_reaction_on_user?
```

Comment:
This should be safe because we don't have a way to create vomit reactions on users anywhere.